hack/bazel-build-images.sh

```diff
@@ -24,23 +24,48 @@ source hack/bootstrap.sh
 source hack/config.sh
 
 # vars are uninteresting for the build step, they are interesting for the push step only
+other_images_default="
+    //cmd/sidecars:sidecar-shim-image
+    //cmd/libguestfs:libguestfs-tools-image
+    //containerimages:alpine-container-disk-image
+    //containerimages:fedora-with-test-tooling
+    //images/disks-images-provider:disks-images-provider-image
+    //images/vm-killer:vm-killer-image
+"
+
+other_images_x86_64_aarch64="
+    //cmd/sidecars/smbios:example-hook-sidecar-image
+    //cmd/sidecars/disk-mutation:example-disk-mutation-hook-sidecar-image
+    //cmd/sidecars/cloudinit:example-cloudinit-hook-sidecar-image
+    //cmd/sidecars/network-slirp-binding:network-slirp-binding-image
+    //cmd/sidecars/network-passt-binding:network-passt-binding-image
```

Comment:
`network-passt-binding-cni-image` is missing

Reply:
I don't see this image on the main branch yet?

Reply:
Ahh its here - [cmd](https://github.com/kubevirt/kubevirt/tree/main/cmd)/[cniplugins](https://github.com/kubevirt/kubevirt/tree/main/cmd/cniplugins)

Reply:
Added the missing image.